### DISCUSSION TITLE
List of name/value pairs with name aligned left and value aligned right

### DISCUSSION THREAD

**Discussion by ddrake** [tailwindlabs/tailwindcss#11477] [Help] [answered]
Hi,  I'm mostly a backend developer, so not an expert in CSS, but Tailwind is so great it's inspiring me to do more in the front end!  I want to make a simple view that displays a list of name/value pairs after a form is submitted with one pair on each line.  The names should be aligned left, but since most of the values are numbers, I would like them aligned right.  I'm sure this can be done many ways.  I'm open to any html approach, though perhaps a definition list would be semantically reasonable.  It should look something like this:
```
Equipment Cost             $5,678
Building Depreciation:     $8,345
etc...
```
I appreciate any suggestions.   Thanks!
Dow

**Comment by wongjn** [comment, marked as answer]
Here's an approach using CSS grid: https://play.tailwindcss.com/KEM3FCBRxo  
And another with tables: https://play.tailwindcss.com/k0eywnKZuS

**Reply by ddrake** [reply]
That's great -- thanks wongin!
